Mage/Module.cs

```diff
@@ -90,6 +90,11 @@ public class Module<T> : Singleton<T> where T : class, new() {
 	
 	public void setupUsercommands (Action<Exception> cb) {
 		logger.info ("Setting up usercommands");
+
+		if (commands == null) {
```

Comment:
Is there any chance commands could be not null but empty? The same behavior might make sense. Alternatively, if commands is always allowed to be something so we can initialize our dictionaries, why not throw on null?